echopype/convert/set_groups_ek80.py

```diff
@@ -24,20 +24,13 @@ def set_env(self, env_only=False) -> xr.Dataset:
         ping_time = np.array([ping_time.astype("datetime64[ns]")])
 
         # Collect variables
+        tmp_env = self.parser_obj.environment.copy()
+        dict_env = dict()
+        for k, v in tmp_env.items():
+            if k in ["temperature", "depth", "acidity", "salinity", "sound_speed"]:
+                dict_env[k] = (["ping_time"], [v])
         ds = xr.Dataset(
-            {
-                "temperature": (
-                    ["ping_time"],
-                    [self.parser_obj.environment["temperature"]],
-                ),
-                "depth": (["ping_time"], [self.parser_obj.environment["depth"]]),
-                "acidity": (["ping_time"], [self.parser_obj.environment["acidity"]]),
-                "salinity": (["ping_time"], [self.parser_obj.environment["salinity"]]),
-                "sound_speed_indicative": (
-                    ["ping_time"],
-                    [self.parser_obj.environment["sound_speed"]],
-                ),
```

Comment:
Note that your new, more compact scheme doesn't work for sound speed, b/c the input and output variable names are different: `sound_speed` vs `sound_speed_indicative`.  That is, unless the original code was wrong; but in that case I suggest you make a note that you're also correcting an error.

Reply:
This with the last one clearly showed that I wasn't awake... -_-

It should be `sound_speed_indicative` as defined in the convention. If there's absorption it should be `absorption_indicative`. I'll change that.

Reply:
I'll include this as part of #537.